packages/proxmark3-git/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
+TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="2021.05.10-0d3c2978"
+TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/0d3c2978a6516def4d722cacdb9d651b087df2a1.tar.gz
+TERMUX_PKG_SHA256=702e50af914dbc23ad2bb41de19217dc5408f7ae7181868e005dc1052051355d
+TERMUX_PKG_BUILD_DEPENDS="readline"
```

Comment:
Shouldn't this be depends and not build_depends?

Reply:
Normal `proxmark3` package also specifies `TERMUX_PKG_BUILD_DEPENDS="readline"`. I'm not using this package, so don't know whether `readline` is a strict dependency, maybe there are binaries which not require it.

However `readline` is always-installed package because is a dependency for `bash`.